### PR TITLE
[NETBEANS-4571] Show PHP Static Analysis configuration files in Important Files

### PR DESCRIPTION
**Pull request by KacerCZ**
https://issues.apache.org/jira/browse/NETBEANS-4571

Added standard configuration files of PHP static analysis tools to Important Files.

NETBEANS-4571

### CODE REVIEW

File: php/php.code.analysis/src/org/netbeans/modules/php/analysis/ImportantFilesImpl.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        assert project != null;
+        support = ImportantFilesSupport.create(project.getProjectDirectory(),
+                CodeSniffer.CONFIG_FILE_NAME,
+                CodeSniffer.DOT_CONFIG_FILE_NAME,
+                CodeSniffer.DIST_CONFIG_FILE_NAME,
+                CodeSniffer.DIST_DOT_CONFIG_FILE_NAME,
+                CodingStandardsFixer.CONFIG_FILE_NAME,
+                CodingStandardsFixer.DIST_CONFIG_FILE_NAME,
+                PHPStan.CONFIG_FILE_NAME,
+                PHPStan.DIST_CONFIG_FILE_NAME);
```
Comment (junichi11) [inline php/php.code.analysis/src/org/netbeans/modules/php/analysis/ImportantFilesImpl.java:47]: Nitpick: I would separate these from here. e.g. add the field or add the get method

Comment (KacerCZ) [reply php/php.code.analysis/src/org/netbeans/modules/php/analysis/ImportantFilesImpl.java]: I agree. I moved list of file names to field.

**Comment by tmysik** [comment]
@KacerCZ Looks good to me but please verify the test failures, thanks.

@junichi11 Once everything is OK for you, feel free to merge. Thanks.

**Comment by KacerCZ** [comment]
Build failure was caused by network connection problem and is not related to content of PR.

**Comment by KacerCZ** [comment]
@junichi11 @tmysik Thanks for review.

**Pull request merged**